kuma/static/styles/includes/vars.styl

```diff
@@ -232,3 +232,8 @@ $column-12, $column-all {
     margin: 0;
     float: none;
 }
+
+/*
+edit-mode customizations - applied to blocks only in edit mode
+====================================================================== */
+$editor-box-border = 2px dotted rgba(170, 221, 255, 255);
```

Comment:
If you meant "no translucency", the hex value would be ``#ADF``, which is a light blue.